tests/unittest/executor/test_sleep_collective_rpc_guards.py

```
@@ -0,0 +1,162 @@
+from types import SimpleNamespace
+from unittest.mock import MagicMock
+
+import pytest
+
+
+# ---------------------------------------------------------------------------
+# Shared helpers
+# ---------------------------------------------------------------------------
+
+
+def _make_worker(backend="pytorch", world_size=1, sleep_config=object()):
```

Ruff (0.15.12)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)